docs/docs/guides/installation/get-started/macos.mdx

```diff
@@ -0,0 +1,33 @@
+## Download The CockroachDB binary
+
+Download a `cockroach` binary as described [in the CockroachDB docs](https://www.cockroachlabs.com/docs/v22.1/install-cockroachdb).
+ZITADEL is tested against CockroachDB v22.1.0.
+
+## Run CockroachDB and ZITADEL
+
+Run a CockroachDB instance
+
+```bash
+cockroach start-single-node --insecure --background --http-addr :9090
+```
+
+# Download and Install ZITADEL
+
+Either use `Homebrew`  ...
+
+```bash
+brew install zitadel/tap/zitadel
+```
+
+... or download the binary from GitHub
+
+```bash
+export MY_ARCHITECTURE="arm64 or amd64 depeding on your mac"
+curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0-v2-alpha.24 | grep "browser_download_url.*zitadel_Darwin_${MY_ARCHITECTURE}.tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+```
+
+Run ZITADEL
+
+```bash
+ZITADEL_EXTERNALSECURE=false zitadel start-from-init --masterkey "MasterkeyNeedsToHave32Characters"
+```
```

Comment:
```suggestion
```bash
ZITADEL_EXTERNALSECURE=false zitadel start-from-init --masterkey "MasterkeyNeedsToHave32Characters" --tlsMode disabled
```
```

I followed the instructions and got the following error:
FATA[0000] unable to start zitadel                       caller="/home/runner/work/zitadel/zitadel/cmd/start/start_from_init.go:41" error="TLS is enabled: please specify a key (path) and a cert (path) or disable TLS if needed (e.g. by setting flag `--tlsMode external` or `--tlsMode disabled"